fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/export/system/ChunkReader.java

```diff
@@ -217,8 +217,21 @@ public Object readItem() throws Exception {
         pageNum++;
 
         if (chunkData == null) {
-            chunkData = new TransientUserData(pageNum, null, new ArrayList<PartETag>(), 1, 0, null, 0, 0, 0, 1, 0, 1);
-            chunkData.setLastPageNum(searchContext.getLastPageNumber());
+            chunkData = (TransientUserData)TransientUserData.Builder.builder()
+                    .pageNum(pageNum)
+                    .uploadId(null)
+                    .cosDataPacks(new ArrayList<PartETag>())
+                    .partNum(1)
+                    .indexOfCurrentTypeFilter(0)
+                    .resourceTypeSummary(null)
+                    .totalResourcesNum(0)
+                    .currentUploadResourceNum(0)
+                    .currentUploadSize(0)
+                    .uploadCount(1)
+                    .lastPageNum(searchContext.getLastPageNumber())
+                    .lastWritePageNum(1)
+                    .build();
```

Comment:
This is duplicate of other ChunkReader implementation, do you want to simplify?

Reply:
Group export reader inherits from Patient reader, but System export reader is very different from them, so I didn't really want to mix them...